addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -1728,11 +1754,12 @@ QUnit.test('[technical] message list with a full page of empty messages should s
             id: 11,
             model: 'mail.channel',
         }),
+        order: 'asc',
     });
     await this.afterEvent({
         eventName: 'o-thread-view-hint-processed',
         func: () => {
-            this.createThreadViewComponent(threadViewer.threadView, { order: 'asc' }, { isFixedSize: true });
+            this.createThreadViewComponent(threadViewer.threadView, { }, { isFixedSize: true });
```

Comment:
too bad we are using positional params, but then it can `undefined` instead of `{}`